src/sortStudents.ts

```diff
@@ -1,16 +1,60 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function Average(student: Student): number {
```

Comment:
```suggestion
function getAverageGrades(student: Student): number {
```
Use camelCase. Also, please follow these recommendations on naming functions:
![image](https://user-images.githubusercontent.com/96409563/207949784-0e85a638-23d9-4d71-9e82-e027b5169b62.png)